### DISCUSSION TITLE
v0.6.0-M1

### DISCUSSION THREAD

**Discussion by erikerlandson** [erikerlandson/coulomb#287] [General]
- Milestone release for 0.6.0: the first Scala 3 release for coulomb

<hr /><em>This discussion was created from the release <a href='https://github.com/erikerlandson/coulomb/releases/tag/v0.6.0-M1'>v0.6.0-M1</a>.</em>

**Comment by armanbilge** [comment]
Just synchronized to central! 🚀 
https://repo1.maven.org/maven2/com/manyangled/coulomb-core_3/

**Reply by armanbilge** [reply]
And unidocs are available here:
https://www.javadoc.io/doc/com.manyangled/coulomb-docs_3/latest/index.html

**Reply by erikerlandson** [reply]
That is awesome :tada:
